src/library/Notifications/Wrapper.ts

```diff
@@ -26,20 +26,20 @@ export const Wrapper = styled.ul`
     overflow: hidden;
 
     &.notification--error {
-      background: var(--background-error);
+      background: var(--status-danger-color);
```

Comment:
That's a reassurance that making css vars type-checked in Common was a good idea ;D